source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.test.tsx

```diff
@@ -162,27 +162,6 @@ describe('AcquisitionContainer component', () => {
     expect(canRemoveResponse).toBe(true);
   });
 
-  it('canRemove returns false if file property has associated entities', async () => {
-    const { getByTestId } = setup(undefined, { claims: [] });
-
-    const spinner = getByTestId('filter-backdrop-loading');
-    await waitForElementToBeRemoved(spinner);
-
-    mockAxios.onGet(new RegExp('acquisitionfiles/1/properties')).reply(200, [
-      {
-        id: 1,
-        isDisabled: false,
-        property: {
-          id: 1,
-        },
-        activityInstanceProperties: [{}],
-      },
-    ]);
-    await act(async () => viewProps.onShowPropertySelector());
-    const canRemoveResponse = await act(() => viewProps.canRemove(1));
-    expect(canRemoveResponse).toBe(false);
-  });
-
```

Comment:
why delete an existing (passing) test?

Reply:
There is no longer a functional difference. See the answer below